app/assets/stylesheets/_main.scss

```
@@ -457,3 +457,13 @@ body {
 .govuk-\!-text-wrap-balance {
   text-wrap: balance;
 }
+
+.dashed-box {
+  padding: 30px;
+  border: 1px dashed #B1B4B6;
```

It might be better to use the govuk-colour function rather than the hex code:

```suggestion
  border: 1px dashed govuk-colour("mid-grey");
```